app/src/main/kotlin/cz/covid19cz/erouska/net/Converter.kt

```diff
@@ -0,0 +1,48 @@
+package cz.covid19cz.erouska.net
+
+import com.google.android.gms.common.api.ApiException
+import com.google.android.gms.common.api.Status
+import com.google.gson.Gson
+import cz.covid19cz.erouska.net.model.Data
+import okhttp3.RequestBody
+import okhttp3.ResponseBody
+import retrofit2.Converter
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.io.IOException
+import java.lang.reflect.ParameterizedType
+import java.lang.reflect.Type
+
+class CustomConverterFactory(gson: Gson) : Converter.Factory() {
```

Comment:
Proč vlastní converter? Nestačily by nám gson anotace u dto?

Reply:
Změr je vytvořit více obecné řešení pro případ dalších API callů s stejnou JSON strukturou.

Reply:
To se dá řešit generickou BaseResponse ;-)